zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/OrchestratorZoneService.java

```diff
@@ -179,20 +180,20 @@ private String getZoneUri(String subdomain, String uaaUri) {
         return newUAARoute.toString();
     }
 
+    private String getIssuerId(String subdomain) {
```

Comment:
getZoneUri() and getIssuerId() are very similar. It could be refactored to share the common code.